cmd/list.go

```diff
@@ -3,24 +3,28 @@ package cmd
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/ludviglundgren/qbittorrent-cli/internal/config"
 	"github.com/ludviglundgren/qbittorrent-cli/pkg/qbittorrent"
 
-	"github.com/spf13/cobra"
 	"github.com/dustin/go-humanize"
+	"github.com/spf13/cobra"
 )
 
 // RunList cmd to list torrents
 func RunList() *cobra.Command {
 	var json bool
 
 	var command = &cobra.Command{
-		Use:   "list",
+		Use:   "list [state]",
 		Short: "List torrents",
-		Long:  `List all torrents`,
+		Long: `List all torrents, or torrents with a specific state. Available states:
+all, downloading, seeding, completed, paused, active, inactive, resumed, 
+stalled, stalled_uploading, stalled_downloading, errored`,
+		Example: `qbt list downloading`,
 	}
-	command.Flags().BoolVar(&json, "json", false, "print to json")
+	command.Flags().BoolVar(&json, "json", false, "Print to json")
```

Comment:
While this works, I would add a new Flag like `filter`,`state` or `status` that takes the argument. It would then become `qbt list --filter=downloading` for example.

And then we can simplify the check for args below.

I would add a new `StringVar` Flag [like here](https://github.com/ludviglundgren/qbittorrent-cli/blob/develop/cmd/add.go#L48).

```golang
command.Flags().StringVar(&state, "state", "", "Filter by state: all, downloading, seeding, completed, paused, active, inactive, resumed, 
stalled, stalled_uploading, stalled_downloading, errored")
```